src/chapters/booleans-and-conditionals/exercises.rst

```diff
@@ -1,8 +1,7 @@
 Exercises: Booleans and Conditionals
 ====================================
 
-If your teacher added you to a :ref:`repl.it classroom <replit-classroom-assignments>`
-or a :ref:`Trinket course <trinket-course-assignments>`, complete the exercises
+If your teacher added you to a :ref:`Trinket course <trinket-course-assignments>`, complete the exercises
 there.
 
 Otherwise, use the links below to code in your own free account.
```

Comment:
With the replit reference gone, let's point out that the students can still use that free account as well.

"Otherwise, use the links below to code in your own free Replit or Trinket account."

Reply:
Done